apps/www/pages/_app.tsx

```diff
@@ -66,12 +69,35 @@ export default function App({ Component, pageProps }: AppProps) {
 
   const forceDarkMode = pathname === '/' || router.pathname.startsWith('/launch-week')
 
+  let applicationName = 'Supabase'
+  let faviconRoute = DEFAULT_FAVICON_ROUTE
+  let themeColor = DEFAULT_FAVICON_THEME_COLOR
+
+  console.log('router.asPath', router.asPath)
+
+  switch (router.asPath) {
+    case '/launch-week':
+      applicationName = 'Supabase LWX'
+      faviconRoute = 'images/launchweek/lwx/favicon/'
+      themeColor = 'FFFFFF'
+      break
+    default:
+      break
+  }
+
```

Comment:
@fsansalvadore make note of new strategy for setting favicons.
It didn't seem to replace some of the favicons when using `<MetaFaviconsPagesRouter/>` later down the page tree.